console/console-init/ui/src/modules/address/components/AddressToggleGroup/AddressToggleGroup.tsx

```diff
@@ -96,15 +96,21 @@ const AddressToggleGroup: React.FunctionComponent<IAddressToggleGroupProps> = ({
 
   const toggleItems = (
     <>
-      <ToolbarItem spacer={{ md: "spacerNone" }} data-codemods="true">
+      <ToolbarItem
+        spacer={{ md: "spacerNone" }}
+        data-codemods="true"
+        id="addr-toggle-group-name-toolbaritem"
```

Comment:
Can we revisit the id name here as well please, as it's unclear what this component exactly does on the UI?